.tekton/osc-podvm-builder-pull-request.yaml

```diff
@@ -96,11 +98,11 @@ spec:
       description: Image tag expiration time, time values could be something like
         1h, 2d, 3w for hours, days, and weeks, respectively.
       name: image-expires-after
-    - default: "false"
+    - default: "true"
       description: Build a source image.
       name: build-source-image
       type: string
-    - default: "true"
+    - default: "false"
```

Comment:
This one needs to stay "true" - we did not modify it, and the new pipeline requires it to be "true" because it relies on OCI artifacts to share files between the different tasks